src/Servers/Kestrel/Transport.MsQuic/src/Internal/MsQuicConnection.cs

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Connections.Features;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic.Internal
+{
+    internal class MsQuicConnection : TransportConnection, IStreamListener
+    {
+        private QuicConnection _connection;
+        private IAsyncEnumerator<MsQuicStream> _acceptEnumerator;
```

Comment:
Get rid of this. It's not needed here, just inline the accept code into the AcceptAsync method itself.